Source/WebCore/Modules/webauthn/WebAuthenticationConstants.h

```diff
@@ -78,7 +78,7 @@ enum class ClientDataType : bool {
     Get
 };
 
-constexpr const char LocalAuthenticatorAccessGroup[] = "com.apple.webkit.webauthn";
+// constexpr const char LocalAuthenticatorAccessGroup[] = "com.apple.webkit.webauthn";
```

Comment:
Keep this and make it `constexpr const CFStringRef LocalAuthenticatorAccessGroup = CFSTR("com.apple.webkit.webauthn");` and use it instead of the inline version you have below.